app/src/main/res/values/strings.xml

```diff
@@ -910,6 +909,8 @@
     <string name="gps_not_enabled">GPS not enabled</string>
     <string name="allow_gps">Allow GPS</string>
     <string name="allow_gps_info">Please turn on GPS to include the current location in the logged file</string>
+    <string name="location_enabled">Location: Enabled ✔</string>
```

Comment:
Will these icons be displayed in every device? Or we can just have only the text without the symbol

Reply:
Yeah I will remove them.